common/Jet_QA.C

```diff
@@ -511,43 +567,40 @@ void JetsWithTracksQA(std::optional<uint32_t> trg = std::nullopt)
 	for (uint32_t jet : vecJetsToQA)
 	{
 
-		// intialize and register sum track vs. jet pt qa module
-		StructureinJets* jetStructQA = new StructureinJets(
-				"StructureInJets" + trig_tag + "_" + JetQA::JetTag[jet],
-				JetQA::JetInput[jet],
-				""
-				);
-		jetStructQA -> Verbosity(verbosity);
-		jetStructQA -> writeToOutputFile(false);
-		if (trg.has_value())
-		{
-			jetStructQA -> setTrgToSelect(trg.value());
-		}
-		se -> registerSubsystem(jetStructQA);
-
-		// initialize and register track jet qa module
-		TrksInJetQA* trksInJetQA = new TrksInJetQA("TrksInJet" + trig_tag + "_" + JetQA::JetTag[jet]);
-		trksInJetQA -> SetHistSuffix("");
-		trksInJetQA -> Configure(
-				{
-				.outMode     = TrksInJetQA::OutMode::QA,
-				.verbose     = Enable::QA_VERBOSITY,
-				.doDebug     = false,
-				.doInclusive = false,
-				.doInJet     = true,
-				.doHitQA     = false,
-				.doClustQA   = true,
-				.doTrackQA   = true,
-				.doJetQA     = true,
-				.rJet        = 0.4,
-				.jetInNode   = JetQA::JetInput[jet]
-				}
-				);
-		if (trg.has_value())
-		{
-			trksInJetQA -> SetTrgToSelect(trg.value());
-		}
-		se -> registerSubsystem(trksInJetQA);
+	    // intialize and register sum track vs. jet pt qa module
+	    StructureinJets* jetStructQA = new StructureinJets(
+	    "StructureInJets" + trig_tag + "_" + JetQA::JetTag[jet],
+	    JetQA::JetInput[jet],"");
+	    jetStructQA -> Verbosity(verbosity);
+	    jetStructQA -> writeToOutputFile(false);
+	    if (trg.has_value())
+	    {
+	     jetStructQA -> setTrgToSelect(trg.value());
+	    }
+	    se -> registerSubsystem(jetStructQA);
+
+	    // initialize and register track jet qa module
+	    TrksInJetQA* trksInJetQA = new TrksInJetQA("TrksInJet" + trig_tag + "_" + JetQA::JetTag[jet]);
+	    trksInJetQA -> SetHistSuffix("");
+	    trksInJetQA -> Configure(
+	    {
+	    .outMode     = TrksInJetQA::OutMode::QA,
+	    .verbose     = Enable::QA_VERBOSITY,
+	    .doDebug     = false,
+	    .doInclusive = false,
+	    .doInJet     = true,
+	    .doHitQA     = false,
+	    .doClustQA   = true,
+	    .doTrackQA   = true,
+	    .doJetQA     = true,
+	    .rJet        = 0.4,
+	    .jetInNode   = JetQA::JetInput[jet]
```

Comment:
Minor suggestion in line with indenting comment above.
```suggestion
	      .outMode     = TrksInJetQA::OutMode::QA,
	      .verbose     = Enable::QA_VERBOSITY,
	      .doDebug     = false,
	      .doInclusive = false,
	      .doInJet     = true,
	      .doHitQA     = false,
	      .doClustQA   = true,
	      .doTrackQA   = true,
	      .doJetQA     = true,
	      .rJet        = 0.4,
	      .jetInNode   = JetQA::JetInput[jet]
```